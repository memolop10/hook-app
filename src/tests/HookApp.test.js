import React from 'react';
import HookApp from '../HookApp'

import '@testing-library/jest-dom'
import { shallow } from 'enzyme';



describe('Test in HookApp', () => {

    test('should render corectly the component HookApp', () => {
        
        let wrapper = shallow( <HookApp /> )
        console.log(wrapper)
        expect( wrapper ).toMatchSnapshot();
       
    })
    
});
