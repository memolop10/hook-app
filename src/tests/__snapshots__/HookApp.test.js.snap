// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in HookApp should render corectly the component HookApp 1`] = `
<div>
  <h1>
    Hola mundo!!!
  </h1>
</div>
`;
